test/testinput/bufr_ncep_adpupa_nc002001.yaml

```diff
@@ -0,0 +1,148 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# 
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+#     obsdatain: "./testinput/gdas.t12z.adpupa_thinned.tm00.bufr_d"
+      obsdatain: "NC002001"
+
+      exports:
+        group_by_variable: pressure
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "NC002001/YEAR"
+              month: "NC002001/MNTH"
+              day: "NC002001/DAYS"
+              hour: "NC002001/HOUR"
+              minute: "*/MINU"
+          stationIdentification:
+            query: "NC002001/RPID"
+          latitude:
+            query: "NC002001/CLAT"
+          longitude:
+            query: "NC002001/CLON"
+          stationElevation:
+            query: "NC002001/SELV"
+
+          # ObsValue
+          pressure:
+            query: "NC002001/UARLV/PRLC"
+          airTemperature:
+            query: "NC002001/UARLV/UATMP/TMDB"
+          dewpointTemperature:
+            query: "NC002001/UARLV/UATMP/TMDP"
+          windDirection:
+            query: "NC002001/UARLV/UAWND/WDIR"
+          windSpeed:
+            query: "NC002001/UARLV/UAWND/WSPD"
+
+          # QualityMarker
+          pressureQM:
+            query: "NC002001/UARLV/QMPR"
+          airTemperatureQM:
+            query: "NC002001/UARLV/UATMP/QMAT"
+          moistureQM:
+            query: "NC002001/UARLV/UATMP/QMDD"
+          windQM:
+            query: "NC002001/UARLV/UAWND/QMWN"
+
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t12z.adpupa_bufr_nc002001.tm00.nc"
+
+      dimensions:
+        - name: RadiosondeReportLevelData
+          path: "*/UARLV"
```

Comment:
Is NC002001 required for this?

Reply:
Has been fixed by using the subset in the export section, as per @rmclaren suggestion.